src/requirements/graph/processor/copy-graph.ts

```diff
@@ -0,0 +1,14 @@
+import GraphProcessor from './interface';
+import RequirementFulfillmentGraph from '..';
+import { CourseWithUniqueId } from '../types';
+
+/**
+ * Essentially a noop, but copies the graph. This can be useful for debugging.
+ */
+export default class CopyGraph<Requirement extends string, Course extends CourseWithUniqueId>
+  implements GraphProcessor<Requirement, Course> {
+  // eslint-disable-next-line class-methods-use-this
+  public process(graph: RequirementFulfillmentGraph<Requirement, Course>) {
+    return graph.copy();
+  }
+}
```

Comment:
Could this be the default implementation of `GraphProcessor` instead of creating a dedicate subclass?